src/components/game/Board.jsx

```diff
@@ -1,26 +1,8 @@
-import React, { useState } from "react";
```

Comment:
Thanks for deleting dead code!

Importing `React` with functional components is no longer needed after React v17 so great catch here.